rosidl_typesupport_cpp/QUALITY_DECLARATION.md

```diff
@@ -2,9 +2,9 @@ This document is a declaration of software quality for the `rosidl_typesupport_c
 
 # rosidl_typesupport_cpp Quality Declaration
 
-The package `rosidl_typesupport_cpp` claims to be in the **Quality Level 4** category.
+The package `rosidl_typesupport_cpp` claims to be in the **Quality Level 3** category.
```

Comment:
This QD needs to be updated to 2. Has that been done in rolling?

Reply:
I will forward port the commit to master